test/helper.js

```diff
@@ -89,6 +87,7 @@ class Helper {
     }
 
     function onconnect () {
+      if (this.logging) console.log('bootpipe connected to sidecar')
```

Comment:
what does this.logging point to?

Reply:
true... this was pointing nowhere... the intention was to point to a property in the class helper, but `this` is not bound here. I simplified it to log always in tests for now